packages/cli/src/commands/doctor/healthchecks/python.ts

```diff
@@ -0,0 +1,55 @@
+import versionRanges from '../versionRanges';
+import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {logManualInstallation} from './common';
+import {HealthCheckInterface} from '../types';
+
+import download from '../../../tools/downloader';
+import {updateEnvironment} from '../../../tools/variables';
+import {join} from 'path';
+import {Ora} from 'ora';
+import {executeCommand} from '../../../tools/executeWinCommand';
+
+export default {
+  label: 'Python',
+  getDiagnostics: async ({Languages}) => ({
+    needsToBeFixed: doesSoftwareNeedToBeFixed({
+      version:
+        (Languages && Languages.Python && Languages.Python.version) || '',
+      versionRange: versionRanges.PYTHON,
+    }),
+
+    version: (Languages && Languages.Python && Languages.Python.version) || '',
+    versionRange: versionRanges.PYTHON,
+  }),
+  win32AutomaticFix: async ({loader}: {loader: Ora}) => {
```

Comment:
I like this but I don't really understand why it's needed.

I assume that Python is not required on other operational systems so, wouldn't this be the same case of `ios-deploy`/`CocoaPods` health check? If the OS running is not the one targeted by the health check then it doesn't run it at all.

Reply:
You are right that it should be the same as `CocoaPods` for this particular case. I put it in the common health checks because I was testing the fallback (i.e.: on Linux the `runAutomaticFix` should be executed instead of `win32AutomaticFix`) and the forgot to change it.

The idea is to make it easy to implement "installers" for those dependencies that are shared across different OS. Does it make more sense now? In https://github.com/react-native-community/cli/issues/976#issuecomment-589882936 there's a bit more information

Also I'll move Python to the Windows only checks. Thanks for catching it up!

Reply:
@lucasbento now Python is only added if the platform is win32

Reply:
Sorry I didn't see #976 before, this is good stuff ❤️